### ISSUE TITLE
Setting alpha reverts the color of a histogram

### ISSUE THREAD

**Issue by jlstevens** [ioam/holoviews#165]
This is probably a bug due to the recent changes in the options system.

```python
import numpy as np
from holoviews import Histogram
np.random.seed(1)
data = [np.random.normal() for i in range(10000)]
frequencies, edges = np.histogram(data, 20)
Histogram(frequencies, edges)(style={'facecolor':'r'})
```

Now if you add ``'alpha':0.5`` to the style, the color reverts to the default blue.

**Comment by jlstevens** [comment]
I've traced it down to the annoying matplotlib aliases. If you use 'fc' it works:

```python
Histogram(frequencies, edges)(style={'fc':'r', 'alpha':0.5})
```

The problem is if you use 'facecolor', this option is added along with the default 'fc' value (the default color cycle)

**Comment by jlstevens** [comment]
Should be fixed in 041234a.
